templates/client/show.html.twig

```diff
@@ -1,40 +1,51 @@
 {% extends 'base.html.twig' %}
-{% block stylesheets %}
-    {{ encore_entry_link_tags('board') }}
-    {{ encore_entry_link_tags('app') }}
-{% endblock %}
-    {% block header %}
-<header>
-    <nav>
-        <a class="logoContainer" href=""><img src="http://magouvernance.fr/wp-content/uploads/2019/05/LOGO-TRANSPARENT.png"  alt="logo"/></a>
+    {% block stylesheets %}
+        {{ encore_entry_link_tags('clientBoard') }}
+        {{ encore_entry_link_tags('app') }}
```

Comment:
use {{ parent() }} instead of loading app every pages ?

Reply:
Resolved